src/solver/simulation/sim_alloc_probleme_hebdo.cpp

```diff
@@ -41,10 +42,27 @@ void SIM_AllocationProblemeHebdo(PROBLEME_HEBDO& problem, unsigned NombreDePasDe
 {
     auto& study = *Data::Study::Current::Get();
 
+    try
+    {
+        SIM_AllocationProblemeDonneesGenerales(problem, study, NombreDePasDeTemps);
```

Comment:
If those functions are only used in that source file, it would be even better to not have them in a header at all.
They should be in this file in the anonymous namespace, to not clutter the list of symbols visible from other compilation units.

Reply:
TBD in a different PR